src/main.rs

```diff
@@ -23,6 +25,10 @@ struct Atdf2SvdOptions {
     #[options(free)]
     svd_path: Option<std::path::PathBuf>,
 
+    /// List of patches to apply.
+    #[options(short = "p", long = "auto-patches")]
```

Comment:
Ah sorry, I should have said this more clearly.  I also meant for the `-p` short variant to be dropped here.

```suggestion
    #[options(long = "auto-patches")]
```

Reply:
Done